lib/cocoapods/installer/target_installer/pod_target_installer.rb

```diff
@@ -262,6 +275,14 @@ def create_module_map
           FileUtils.cp(custom_module_map, path)
           add_file_to_support_group(path)
 
+          unless target.requires_frameworks?
+            File.open(path) do |source_file|
+              contents = source_file.read
+              contents.gsub!(/^\s*framework\s*module/, 'module')
```

Comment:
You can use `sub!` in combination with `\A` in the regex to match the beginning of the string multi-line, in that way we ensure to match only the root declaration.